userspace/libsinsp/parsers.cpp

```diff
@@ -4513,54 +4553,54 @@ void sinsp_parser::parse_container_json_evt(sinsp_evt *evt)
 		sinsp_container_info container_info;
 		const Json::Value& container = root["container"];
 		const Json::Value& id = container["id"];
-		if(!id.isNull() && id.isConvertibleTo(Json::stringValue))
+		if(!id.isNull() && check_is_convertible(id, Json::stringValue, std::string("id")))
```

Comment:
I'm not sure how I feel about creating multiple new strings for every parse. `const char*`?

BTW I hoped we could move the array lookup into the method as well but it's going to have a pretty ugly interface (we'd have to make it a template and specialize for every JSON type)

Reply:
changed to `cont char *`